### PR TITLE
Added .asf.yaml file

### PR DESCRIPTION
**Pull request by merlimat**
Added similar settings as used in main pulsar repo

### CODE REVIEW

**Comment by BewareMyPower** [comment]
Hi Matteo, do you know how to configure `Squash and merge` by default?

image

We also need to enable the GitHub issues in this repo.

**Comment by merlimat** [comment]
That would be from these lines: 

```
  enabled_merge_buttons:
    # enable squash button:
    squash:  true
    # disable merge button:
    merge:   false
    # disable rebase button:
    rebase:  false
```

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  features:
+    # Enable wiki for documentation
+    wiki: true
+    # Enable issues management
+    issues: true
+    # Enable projects for project management boards
+    projects: true
+  enabled_merge_buttons:
+    # enable squash button:
+    squash:  true
```
Comment (BewareMyPower) [inline .asf.yaml:39]: The `Squash and merge` will be the default behavior after this PR is merged?

Comment (merlimat) [reply .asf.yaml:39]: Correct, it will be the only option

**Review by Demogorgon314 — APPROVED** [review]

**Pull request merged**